Services/Common/OJS.Services.Common/Implementations/SubmissionsDistributorCommunicationService.cs

```diff
@@ -116,6 +116,7 @@ private object BuildDistributorSubmissionBody(Submission submission)
                 {
                     MaxPoints = submission.Problem.MaximumPoints,
                     CheckerType = checkerType,
+                    CheckerParameter = submission.Problem!.Checker!.Parameter,
```

Comment:
Maybe `?` instead of `!`?

Reply:
It's redundant, I think that I had forgotten it.